website/docs/premake_WORKING_DIR.md

```diff
@@ -1,4 +1,4 @@
----
+---control +fhift +m
```

Comment:
Is this the change you intended to submit? Not really sure why we would want this?